### PR TITLE
Schema sorted load

### PR DESCRIPTION
**Pull request by allenmcao**
This PR also contains some commits from #4, so please wait until that has been merged before merging this one.

### CODE REVIEW

File: load_data.py [hunk 1]
```diff
@@ -118,7 +118,7 @@ def load_article(article):
     print 'loading article...'
 
 def load_schema_dir(dirpath):
-    for schema_file in os.listdir(dirpath):
+    for schema_file in sorted(os.listdir(dirpath)):
```
Comment (stefanv) [inline load_data.py:121]: How about `glob.glob('*.txt')`

Comment (stefanv) [inline load_data.py:121]: Not important...

File: webpack/dev-server.js [hunk 2]
```diff
@@ -12,7 +12,7 @@ var config = require('./dev.config');
 var compiler = webpack(config.webpack);
 var devServer = new WebpackDevServer(compiler, config.server.options);
 
-devServer.listen(config.server.port, 'localhost', function () {
+devServer.listen(config.server.port, '0.0.0.0', function () {
```
Comment (btburton42) [inline webpack/dev-server.js:15]: no need for this anymore as it can be passed in as a config var https://github.com/Goodly/TextThresher/pull/13

**Comment by btburton42** [comment]
closing as this work is covered elsewhere 
https://github.com/Goodly/TextThresher/pull/11

**Pull request closed without merging**